test/e2e/backup-resource/deletion.go

```diff
@@ -0,0 +1,162 @@
+/*
+Copyright the Velero contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package backupresource
```

Comment:
There is already a package called `backup` [here](https://github.com/vmware-tanzu/velero/tree/main/test/e2e/backup), so what's different between the `backup` and `backupresource`? It's confusing.